### ISSUE TITLE
Finally moving from ttrss to emacs :) one Q on mark as read?

### ISSUE THREAD

**Issue by zeltak** [areina/elfeed-cljsrn#8]
HI

thx so much for this! i can finally move from ttrss to emacs :) 

I have one question on marking as read from android that will reflect in the elfeed server which i cant quite understand.  how does one mark entries as `read`? i cant seem to swipe to any direction as instructed in the documentation and dont see a mark as read button?

in addition i would love if possible to mark as read if i scroll on the phone past entries (many rss clients on android including the ttrss one do that)

thx again for your great work!

Z

**Comment by areina** [comment]
Hello @zeltak!
You can select an entry doing a long press in an entry row. It's also possible select multiple entries. Then, an icon button should appear in the top bar, where you can press and mark entries as read/unread. 

Swiping right the row is temporarily disabled due to a bug in React Native. Will enable again when fixed. 
Also, if you visit the entry detail, this would be marked as read.

About your comment:

> in addition i would love if possible to mark as read if i scroll on the phone past entries (many rss clients on android including the ttrss one do that)

I don't really understand this behaviour. I think mark entries as read only scrolling down the list is a bit "agressive". Maybe I'm missing something.

Thank you for your feedback, I'm happy to see the app is useful for you and you are moving to Emacs :-)

**Comment by zeltak** [comment]
thx so much @areina for the answer

i see now with the long press cool :)

i do wonder how do you then mark as "read" all the files you go through on the app in order to let the server know you have read them on the phone? say you have 400 entries to you long press each entry one by one and then press read? im probably missing something?

i personally never had any issues with the scroll down and mark as read and have been using it close to 3 years in the ttrss client in android. I find that once in a blue moon that if id pass the entry and want to keep it unread i can always go back and mark it manually as unread

would be very happy to help out in anyway needed (documentation, testing) though keep in mind i cant code to save my life ;-)

best regards

Z

**Comment by areina** [comment]
Oh, right, there is no way to mark all entries as read with a single action. That's because in my daily workflow, probably there are 10 or 15 new entries per day, and I don't need a bulk action mechanism. If it's something specific, I would recommend do it through Emacs, as I see this client as a complement of Elfeed in Emacs and not as an alternative. If it's something that you need for your daily workflow, I could try to add a way to do it.
About the scroll down, it's the same: due to my use case, I have few entries and I read almost all of them, so I would say it isn't useful for me. But I can understand it can be useful for other use cases.
 
In conclusion, I'm going to create two issues to track these features. Would be awesome if other users give their feedback, because then I could see if it's worth developing them or not. Until this moment, with no users feedback, I've developed only what I needed.

**Comment by zeltak** [comment]
thx again! appreciate your answer and time :)

yes i agree its a complement for sure, yet sometimes when working on the road when away from main pc it could be a nice way to pass time at airports etc :)

thx for opening the issues, ill wait to see what other users think  :)

best regards

Z
